js/submit.js

```diff
@@ -0,0 +1,69 @@
+import {mainPin, MAP_INITIAL_COORDS, L} from './map.js';
+import {offerForm, mapFiltersForm} from './form.js';
+
+
+const main = document.querySelector('main');
+const messageOnSuccessTemplate = document.querySelector('#success').content.querySelector('.success');
+const messageOnFailTemplate = document.querySelector('#error').content.querySelector('.error');
+const messageOnSuccess = messageOnSuccessTemplate.cloneNode(true);
+const messageOnFail = messageOnFailTemplate.cloneNode(true);
```

Comment:
Дополнительные одноразовые переменные усложняют чтение кода
```suggestion
const messageOnSuccess = document.querySelector('#success').content.querySelector('.success').cloneNode(true);
const messageOnFail = document.querySelector('#error').content.querySelector('.error').cloneNode(true);
```